x-pack/plugins/cases/public/common/use_cases_toast.tsx

```diff
@@ -161,7 +147,7 @@ export const useCasesToast = () => {
         return toasts.addSuccess({
           color: 'success',
           iconType: 'check',
-          title: toMountPoint(<Title>{renderTitle}</Title>),
+          title: toMountPoint(<TruncatedText text={renderTitle} />),
```

Comment:
Because of my [other comment](https://github.com/elastic/kibana/pull/178379/files#r1524991316) on the `LINE_CLAMP` change I went and double-checked this one. 

It was being used for the success toast only on `useCasesAddToExistingCaseModal` and `useCasesAddToNewCaseFlyout` which happens when trying to attach a dashboard to a case.

Here the `TruncatedText` seems to work fine 👍 

<img width="1229" alt="Screenshot 2024-03-14 at 16 14 58" src="https://github.com/elastic/kibana/assets/1533137/dd1d0b46-a8c0-46a4-873c-6f1568e9d6c1">